unyt/tests/test_unyt_array.py

```diff
@@ -39,6 +31,15 @@
 from packaging.version import Version
 
 from unyt import K, R, Unit, degC, degF, delta_degC, delta_degF, dimensions
+
+# ----------------------------------------------------------------------------
+# Copyright (c) 2013, yt Development Team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
```

Comment:
Can you move this header back to the top of the file?

Reply:
oh, good catch. I think this was done by `isort` or something, but I should have seen it